components/autocomplete/index.js

```diff
@@ -249,9 +249,17 @@ export class Autocomplete extends Component {
 	search( event ) {
 		const { open: wasOpen } = this.state;
 		const { completers } = this.props;
-		// ensure that the cursor location is unambiguous
 		const container = event.target;
+		// check that the event came from a contentEditable
+		// Note that the hasAttribute/getAttribute is because JsDOM does not support container.contentEditable
+		if ( ! container.hasAttribute( 'contentEditable' ) || container.getAttribute( 'contentEditable' ) === 'false' ) {
```

Comment:
Is it that we want to check that the input event occurred on the rendered result of `children` ? We could consider adding a `ref` to it (perhaps with `cloneElement`) then checking that `findDOMNode( ref ) === container`.

I'm not so sure about testing the contenteditable attribute. What if the URL input was refactored to render as a contenteditable?

Reply:
The Autocomplete only works for contentEditable because it uses the selection and calculates node ranges so it makes sense to filter the incoming events to those that target contentEditable.

The tinymce contentEditable is nested within the Editable so checking ```findDOMNode( ref ) === container``` will not work.

I can think of two ways of handling the hypothetical situation of the URL input being refactored as an Editable.

1. We could all the autocomplete to take a selector prop and use Element#matches ( https://developer.mozilla.org/en-US/docs/Web/API/Element/matches ) 
2. We could catch the input events inside Editable and call stopPropogation on any events not from the Editable's TinyMCE.